sources/packages/backend/apps/db-migrations/src/sql/Restrictions/Update-legacy-restrictions-impacts.sql

```diff
@@ -0,0 +1,77 @@
+-- Remove the SSRN legacy map since it will become a direct map to the SIMS restriction.
+DELETE FROM
+    sims.sfas_restriction_maps sfas_restriction_map
+WHERE
+    sfas_restriction_map.code = 'LGCY_SSRN';
+
+-- Rename the existing legacy SSRN converting it to a non-legacy restriction.
+UPDATE
+    sims.restrictions restrictions
+SET
+    restriction_code = 'SSRN',
+    description = 'Poor Scholastic Standing.',
+    restriction_category = 'Academic',
+    notification_type = 'Error',
+    is_legacy = false,
+    action_type = ARRAY ['Stop full time disbursement', 'Stop full time apply'] :: sims.restriction_action_types [],
+    updated_at = NOW(),
+    modifier = (
+        SELECT
+            id
+        FROM
+            sims.users
+        WHERE
+            -- System user.
+            user_name = '8fb44f70-6ce6-11ed-b307-8743a2da47ef@system'
+    )
+WHERE
+    restriction_code = 'LGCY_SSRN';
+
+-- Remap LGCY_SSD to SSR.
+-- Delete the legacy LGCY_SSD map, if exists to ensure the insert can be executed.
+DELETE FROM
+    sims.sfas_restriction_maps
+WHERE
+    code = 'LGCY_SSD';
```

Comment:
Minor. the query is used here slightly different from line 2-4. Without alias. We can follow one way for consistency.

Reply:
Removing the aliases, they were kept by mistake.